src/amo/pages/ErrorPages/ServerErrorPage/index.js

```diff
@@ -0,0 +1,18 @@
+/* @flow */
+import * as React from 'react';
+import NestedStatus from 'react-nested-status';
+
+import ServerError from 'amo/components/Errors/ServerError';
+import Page from 'amo/components/Page';
+
+const ServerErrorPage = () => {
```

Comment:
Hmm. If you go to http://localhost:3000/en-US/firefox/simulate-client-error/ and trigger a render error, it's not rendering the right component (it's happening in [`ErrorPage`](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/App/index.js#L199) which has an error boundary). Here's a fix just to illustrate the issue but I'm not totally sure what to do here.

```diff
diff --git a/src/amo/utils/errors.js b/src/amo/utils/errors.js
index 9efc9d984..e977b1e3d 100644
--- a/src/amo/utils/errors.js
+++ b/src/amo/utils/errors.js
@@ -2,6 +2,7 @@
 import NotAuthorized from 'amo/components/Errors/NotAuthorized';
 import NotFound from 'amo/components/Errors/NotFound';
 import ServerError from 'amo/components/Errors/ServerError';
+import ServerErrorPage from 'amo/pages/ErrorPages/ServerErrorPage';
 
 export function getErrorComponent(status: number | null) {
   switch (status) {
@@ -11,6 +12,6 @@ export function getErrorComponent(status: number | null) {
       return NotFound;
     case 500:
     default:
-      return ServerError;
+      return ServerErrorPage;
   }
 }
```

Reply:
I had commented on this at https://github.com/mozilla/addons-frontend/pull/8858#issue-334328359. I wasn't sure if `getErrorComponent` should return the original component or the new page-level component. The patch above suggests it should be returning the page-level component, so I'll give that a try.